deployment.yaml

```diff
@@ -38,6 +38,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: RoleBinding
 metadata:
   name: placement-policy-binding
+  namespace: placement-controller
```

Comment:
Note that I do this in the README.md via: `kubectl config set-context --current --namespace="${K8S_NAMESPACE}"`

I had it here originally, but then the instructions need to include the need to modify `deployment.yaml`. I'm good either way, but this one will need those extra instructions.